neml/drivers.py

```diff
@@ -596,7 +596,212 @@ def strain_cyclic(model, emax, R, erate, ncycles, T = 300.0, nsteps = 50,
       "energy_density": np.array(ecycle), "plastic_work": np.array(pcycle),
       "history": driver.stored_int[-1], "time": np.array(time)}
 
-def strain_cyclic_followup(model, emax, R, erate, ncycles, 
+def strain_cyclic_extrapolated(model, emax, R, erate, ncycles, T = 300.0, nsteps = 50,
+    sdir = np.array([1,0,0,0,0,0]), hold_time = None, n_hold = 25,
```

Comment:
Add a description to hold_time.
Also, I see this is not a scalar but an array with two components, please document what they are

Reply:
hold time follows from previous cyclic format. it can be a scalar, or a list.